test_add_capacity_node_restart failing with timeout error while waiting for node restart

Test case failing -tests/manage/z_cluster/cluster_expansion/test_add_capacity_with_node_restart.py::TestAddCapacityNodeRestart::test_add_capacity_node_restart[1-workload_storageutilization_rbd0]

The test case is failing due to timeout while waiting for the restarted node to reach Ready state. The test case is passing timeout value as 420 seconds instead of using the default platform specific timeout given in ```restart_nodes``` function. 

```
>       nodes.restart_nodes(nodes=node_list, wait=True, timeout=420)

tests/manage/z_cluster/cluster_expansion/test_add_capacity_with_node_restart.py:70: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
ocs_ci/ocs/platform_nodes.py:1804: in restart_nodes
    node_names=node_names, status=constants.NODE_READY, timeout=timeout
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

node_names = ['sgatfane-21-3az-9w6cl-worker-eastus1-24hrz'], status = 'Ready'
timeout = 420
```
```
        except TimeoutExpiredError:
            log.error(
                f"The following nodes haven't reached status {status}: "
                f"{nodes_not_in_state}"
            )
            raise exceptions.ResourceWrongStatusException(
>               node_names, [n.describe() for n in get_node_objs(node_names)]
            )
E           ocs_ci.ocs.exceptions.ResourceWrongStatusException: Resource ['sgatfane-21-3az-9w6cl-worker-eastus1-24hrz'] describe output: ['Name:  
```

--- a/tests/manage/z_cluster/cluster_expansion/test_add_capacity_with_node_restart.py
+++ b/tests/manage/z_cluster/cluster_expansion/test_add_capacity_with_node_restart.py
@@ -67,7 +67,7 @@ class TestAddCapacityNodeRestart(ManageTest):
         # Restart nodes while additional storage is being added
         logging.info("Restart nodes:")
         logging.info([n.name for n in node_list])
-        nodes.restart_nodes(nodes=node_list, wait=True, timeout=420)
+        nodes.restart_nodes(nodes=node_list, wait=True)
         logging.info("Finished restarting the node list")
 
         # The exit criteria verification conditions here are not complete. When the branch

